src/components/victory-candlestick/helper-methods.js

```diff
@@ -110,18 +110,21 @@ export default {
     }
   },
 
-  getDataStyles(datum, style, props) {
+  getDataStyles(datum, style, props) { // eslint-disable-line complexity
```

Comment:
In this case, I think you should try to refactor rather than ignore this complexity linting error